ecommerce_integrations/woocommerce/doctype/woocommerce_setting/woocommerce_setting.py

```diff
@@ -75,6 +79,9 @@ def create_webhook_url(self):
 		delivery_url = server_url + endpoint
 		self.endpoint = delivery_url
 
+	def migrate_exisiting_items(self):
```

Comment:
this changes are in "test coverage" commit, ideally should be part of the patch commit. Can you rewrite history to edit this.  😅

Reply:
Done